src/components/Posts.tsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect } from 'react';
+import * as postsActions from '../features/posts';
+import { PostsList } from './PostsList';
```

Comment:
There is no import for `Post` type. Make sure all types used in code are properly imported.